components/sidebar/channel_create.jsx

```diff
@@ -52,6 +53,9 @@ export default class ChannelCreate extends React.PureComponent {
             >
                 <button
                     id='createPublicChannel'
+                    role='navigation'
```

Comment:
Yes, you are right, however, since the button sits within the `application` role, it is giving an extra readout, which is why I have used navigation to make it only read it's `aria-label`.
Role `presentation`, or `button` still have that extra readout that I do not want.

Reply:
However, I think you are right, we should rather not do that. We can think about other alternatives later. So I've removed it for now.